guido/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+
+    <link rel="stylesheet" href="./assets/style/style.css">
+    <title>Paella</title>
+
+</head>
+<body>
+<header class="header">
+    <div class="text-heading">
+        <h1 class="primary-heading">
+            <span class="primary-heading-main">paella</span>
+        </h1>
+    </div>
+</header>
+
+<div class="container">
+
+    <h1>
+        Hello, welcome to the most delicious paella recipe!!
+    </h1>
+    <h1 class="heading">Ingredients</h1>
+    <h2 class="heading-secondary">Vegies</h2>
+    <div class="displayVegies"></div>
+    <h2 class="heading-secondary">Land Meats</h2>
+    <div class="displayLandMeats"></div>
+
+    <h1 class="heading">Instructions</h1>
+    <div class="displayInstructions">Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima nemo eveniet quisquam nihil porro. Quam officiis repudiandae dolorem, odit, tempora numquam ex excepturi fugiat iure vel deserunt enim in cum.</div>
+</div>
+
+
+<script src="./assets/js/script.js"></script>
+</body>
+</html>
```

Comment:
so far I enjoyed taking a look at your code, I like some of fun things your adding to your webpage. I didn't find much of anything to critic in your code, so I guess keep going - keep up the good work!!